src/widgets/allergies/allergy-record.component.tsx

```diff
@@ -109,30 +115,28 @@ export default function AllergyRecord(props: AllergyRecordProps) {
             </div>
           </SummaryCard>
           <RecordDetails>
-            <div className={styles.allergyCard}>
-              <table
-                className={`${styles.allergyTable} ${styles.allergyDetails}`}
-              >
-                <thead className="omrs-type-body-regular">
-                  <tr>
-                    <th>Last updated</th>
-                    <th>Last updated by</th>
-                    <th>Last updated location</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  <tr>
-                    <td>
-                      {allergy.lastUpdated
-                        ? dayjs(allergy.lastUpdated).format("DD-MMM-YYYY")
-                        : "-"}
-                    </td>
-                    <td>{allergy.recordedBy}</td>
-                    <td>-</td>
-                  </tr>
-                </tbody>
-              </table>
-            </div>
+            <table
+              className={`${styles.allergyTable} ${styles.allergyDetails}`}
```

Comment:
cc @brandones

Reply:
> cc @brandones

PR description updated to include this fix.